### PR TITLE
Remove some needless imports

### PR DESCRIPTION
**Pull request by jimmywarting**
Attempt at removing lodash

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -105,8 +102,9 @@ const globFiles = async (patterns, {ignores, extensions, cwd}) => (
 const getConfig = async options => {
 	const {options: foundOptions, prettierOptions} = mergeWithFileConfig(normalizeOptions(options));
 	options = buildConfig(foundOptions, prettierOptions);
```
Comment (sindresorhus) [inline index.js:104]: Seems like you can just restructure directly here.

File: index.js [hunk 2]
```diff
@@ -105,8 +102,9 @@ const globFiles = async (patterns, {ignores, extensions, cwd}) => (
...
-	const engine = new ESLint(omit(options, LINT_METHOD_OPTIONS));
-	return engine.calculateConfigForFile(options.filePath);
+	const {filePath, warnIgnored, ...opts} = options;
+	const engine = new ESLint(opts);
```
Comment (sindresorhus) [inline index.js:106]:
```suggestion
	const engine = new ESLint(eslintOptions);
```

File: index.js [hunk 3]
```diff
@@ -117,15 +115,16 @@ const lintText = async (string, inputOptions = {}) => {
 		throw new Error('The `ignores` option requires the `filePath` option to be defined.');
 	}
 
-	const engine = new ESLint(omit(options, LINT_METHOD_OPTIONS));
+	const {filePath, warnIgnored, ...opts} = options;
+	const engine = new ESLint(opts);
```
Comment (sindresorhus) [inline index.js:119]:
```suggestion
	const engine = new ESLint(eslintOptions);
```

File: index.js [hunk 4]
```diff
@@ -3,17 +3,14 @@ const path = require('path');
...
-const pick = require('lodash/pick');
-const omit = require('lodash/omit');
 const micromatch = require('micromatch');
 const arrify = require('arrify');
 const pReduce = require('p-reduce');
 const pMap = require('p-map');
 const {cosmiconfig, defaultLoaders} = require('cosmiconfig');
 const defineLazyProperty = require('define-lazy-prop');
 const pFilter = require('p-filter');
-const {CONFIG_FILES, MODULE_NAME, DEFAULT_IGNORES, LINT_METHOD_OPTIONS} = require('./lib/constants');
```
Comment (sindresorhus) [inline index.js:16]: LINT_METHOD_OPTIONS is now unused and can be removed entirely.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (jimmywarting) [reply index.js]: Not sure what you meant, but did something

**Comment by jimmywarting** [comment]
You have also the option to specify redirect in Request options
```js
fetch(url, {
  redirect: 'manual'
})
```

**Pull request merged**